client/app.html

```diff
@@ -15,6 +15,7 @@
     <script src="../assets/questions/parens.js"></script>
     <script src="../assets/questions/reverseWords.js"></script>
     <script src="../assets/questions/rle.js"></script>
+    <script src="../assets/questions/aad.js"></script>
```

Comment:
Sorry that I didn't check your email carefully. And thank you for your comments. I'll fix these as soon as possible